frontend/individual_pedigree.tsx

```diff
@@ -0,0 +1,160 @@
+      super();
+      this.network = {};
+      this.appRef = createRef();
+    }
+
+    componentDidMount() {
+      this.network = new Network(this.appRef.current, pedigree, this.options);
+      this.network.on("doubleClick", this.onNodeClick)
+    }
+
+    onNodeClick(params) {
+      if (params.nodes.length > 0) {
+        var nodeid = params.nodes[0];
+        window.location.replace("/pedigree/" + nodeid + "/" + generations_input);
+      }
+    }
+
+    render() {
+
+      return (
+        <div ref={this.appRef}  >
+        </div>
+      );
+    }
+  }
+
+  return <>
+    {individual && pedigree && <>
+      <table width="100%">
+        <tbody>
+          <tr>
+            <td width="15%" style={{ verticalAlign: "top" }}>
+              <h2>{individual.name ?? 'unnamed'}</h2>
+              <dl>
+                <dt>Inavelskoefficient</dt> <dd><b>{individual.inbreeding}%</b></dd>
+                <dt>Generationer</dt> <dd><input value={generations_input} onChange={event => setGenerations(event.target.value)} type="number" min="1" max="50"/></dd>
+                <dt>Nummer</dt> <dd>{individual.number}</dd>
+                <dt>Certifikat</dt> <dd>{individual.certificate}</dd>
+                <dt>Kön</dt> <dd>{individual.sex ?? 'unknown'}</dd>
+                <dt>Födelsedatum</dt> <dd>{individual.birth_date ?? '-'}</dd>
+                <dt>Dödsdatum</dt> <dd>{individual.death_date ?? '-'}</dd>
+                <dt>Dödsanteckning</dt> <dd>{individual.death_note ?? '-'}</dd>
+                <dt>Besättning</dt>
+                <Link to={`/herd/${individual.herd.id}`}>
+                  <dd>{individual.herd.name ?? individual.herd.id}</dd>
+                </Link>
+                <dt>Mor</dt>
+                {individual.mother
+                  ? <Link to={`/individual/${individual.mother.id}`}>
+                    <dd>{individual.mother.name}</dd>
+                  </Link>
+                  : <dd>-</dd>
+                }
+                <dt>Far</dt>
+                {individual.father
+                  ? <Link to={`/individual/${individual.father.id}`}>
+                    <dd>{individual.father.name}</dd>
+                  </Link>
+                  : <dd>-</dd>
+                }
+                <dt>Kull</dt> <dd>{individual.litter ?? '-'}</dd>
+                <dt>Färg</dt> <dd>{individual.colour ?? '-'}</dd>
+                <dt>Färgkommentar</dt> <dd>{individual.colour_note ?? '-'}</dd>
+                <dt>Anteckningar</dt> <dd>{individual.notes ?? '-'}</dd>
+                <dt>Vikter</dt>
+                <dd>
+                  {individual.weights.length > 1
+                    ? individual.weights.map((w: any) => `${w.date.substring(0, 16)}: ${w.weight}`).join(", ")
+                    : '-'
+                  }
+                </dd>
+
+              </dl>
+            </td>
+            <td width="85%" >
+              <PedigreeNetwork />
+            </td>
+          </tr>
+        </tbody>
+      </table>
```

Comment:
Since we're using material-ui for the styles in most of the app - consider using their tables ([here](https://material-ui.com/components/tables/)) to get a consistent look on the page easily! :)

Reply:
ok, thanks. I will take a look at that.

Reply:
solved